e2e/filters.spec.ts

```diff
@@ -18,6 +18,9 @@ describe("Filters", () => {
 
     await filters.locator("label").first().waitFor({ timeout: 30_000 });
 
+    // Give some time for the filters to load
+    await sleep(4_000);
```

Comment:
Can we wait for all the filters using the same kind of technique as above (waiting for the labels ?) ?

Reply:
We would need to adjust the logic, as the filter elements are visible, but they can have empty value during loading, so we'd need to wait until every input element has non empty value 👍

I can to try do it, any tips are welcome if you already have the approach in mind, not sure yet how to solve this 🙏

Reply:
I think I managed to do it in https://github.com/visualize-admin/visualization-tool/pull/1387/commits/d73e35602def168e38f095d5d413c47649c7e6ac